service-front/module/Application/src/Controller/Authenticated/Lpa/CheckoutController.php

```diff
@@ -36,10 +38,10 @@ public function indexAction()
         $isRepeatApplication = ($lpa->repeatCaseNumber != null);
 
         $lowIncomeFee = Calculator::getLowIncomeFee($isRepeatApplication);
-        $lowIncomeFee = (floor($lowIncomeFee) == $lowIncomeFee ? $lowIncomeFee : money_format('%i', $lowIncomeFee));
+        $lowIncomeFee = number_format(floatval($lowIncomeFee), 2);
```

Comment:
Compare with what happens in live.

Reply:
Having looked at this and thought about it, the original code will format the cost with decimal point if it has one; otherwise it shows the cost without a decimal point, e.g.

```
cost == 42, display £42
cost == 30.50, display £30.50
```

That's why the calls to `floor()` are there, to see whether the cost has a pence component.